### PR TITLE
`Development`: Add videos to documentation of Exam Timeline and Suspi…

### PR DESCRIPTION
**Pull request by tobias-lippert**
…cious Behavior

#### General

- [x] This is a small issue that I tested locally and was confirmed by another developer on a test server.
- [ ] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).
#### Client
- [x] **Important**: I implemented the changes with a very good performance, prevented too many (unnecessary) REST calls and made sure the UI is responsive, even with large data.
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/).
- [x] Following the [theming guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-design/), I specified colors only in the theming variable files and checked that the changes look consistent in both the light and the dark theme.
- [ ] I added multiple screenshots/screencasts of my UI changes.
- [x] I translated all newly inserted strings into English and German.
### Motivation and Context

Exam Timeline and suspicious behavior documentation lacks screencasts

### Description
Added screencast for both features and a link in the suspicious behavior dashboard to the docs.
### Steps for Testing
Code Review
Make sure you can watch the videos in the docs.
Watch the screencast to verify that the link to the docs is working.

#### Performance Review
- [ ] I (as a reviewer) confirm that the client changes (in particular related to REST calls and UI responsiveness) are implemented with a very good performance 
#### Code Review
- [ ] Code Review 1
- [ ] Code Review 2
#### Manual Tests
- [ ] Test 1
- [ ] Test 2

### CODE REVIEW

**Review by Hialus — CHANGES_REQUESTED** [review]
Looked through the videos and found some general things:
1. You should use personas or TUM accounts for the documentation, not test users. Currently you are logged in as a test user in the videos and have test user 1 as a participant in the exam.
2. The first video is a bit hard to follow, as it is quite fast-paced and I had to pause the video to get a look at what is being shown
3. At the end of the first video you show an empty plagiarism detection screen. I am not sure what the goal here is.
4. Optional suggestion: In video 2 the git-diff loads for quite a while. Might make sense to edit this, so it is a bit faster
5. Optional suggestion: In my opinion the videos would be a lot easier to understand if there was some kind of narration, be it voice or text based. This is obviously a lot of extra work, but it would be nice.
6. Suggestion if you actually reshot the video: Close the other browser tabs, hide the bookmarks and use artemis.cit.tum.de as the domain 😉. You could also just cut off the browser bar in post-production.

Let me know what you think of my suggestions 😄

**Comment by tobias-lippert** [comment]
@Hialus Thanks for the feedback :)
1. I've already removed the test user as exam participant but wasn't aware that I'm also logged in as one --> gonna change that
2. gonna stay longer at the different screens when I redo it.
3. I missed that, that's definitely not intended --> change it
4. should be easy to do --> change it
5. I will have a look if it's possible to add some textual explanation without too much effort.
6. Gonna do that.

**Review by Hialus — APPROVED** [review]
Videos look a lot better now. Great job 👏

**Review by Hialus — APPROVED** [review]

**Review by Strohgelaender — APPROVED** [review]
lgtm, the video links work.
